### ISSUE TITLE
MethodScript.com Accessibility - Images should have alternative text (alt tag)

### ISSUE THREAD

**Issue by LeigerGaming** [EngineHub/CommandHelper#1272]
If an image is important or relevant to the page content, an `alt` tag should be provided with a description of what is happening in the image for the benefit of those that cannot see it. For example, the visually impaired using a screen reader.

When the image is not important (e.g. in this case, it's a search icon), an empty `alt` tag will suffice. Without an empty tag, a screen reader will read out loud the image name, e.g. "search icon dot p n g". Adding an empty alt tag tells a screen reader to ignore it.

There may be other cases, on other pages, that require more fine-tuned fixes. But I don't think there's many images on the website so it should be mostly just this search image that is affected.

```html
<img class="sb-icon-search-normal" src="https://methodscript.com/docs/3.3.4/resources/images/search-icon.png?v=8475693aa4e3368d98bfc4b3872046ce">
```

> All images within a page must be given an alternate text equivalent. Text equivalents for non-text elements communicate the meaning of images to users who cannot perceive the image such as users of screen readers. Proper equivalents provide text which contributes the same level of understanding to the content of the page as the image itself. In instances where the image does not contribute to the understanding of the content and is purely decorative, it needs to be marked in a way to indicate its purely decorative purpose.

References:
- https://www.webaccessibility.com/results/?url=https%3A%2F%2Fmethodscript.com%2Fdocs%2F3.3.4%2Findex.html

**Comment by LeigerGaming** [comment]
Also, images should provide informative alternate text. For example, this is not good! :)

```html
<img alt="Image:CommandHelper_Logo_New.png" src="https://methodscript.com/docs/3.3.4/resources/images/CommandHelper_Logo_New.png" class="maxWidth100Percent">
```

Those poor, poor screen reader users!
